src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -211,14 +211,18 @@ public int[] getRequiredJavadocTokens() {
     @Override
     public void visitJavadocToken(DetailNode ast) {
         final Optional<DetailNode> inlineTag = getInlineTagNode(ast);
-        final DetailNode inlineTagNode = inlineTag.orElse(null);
-        if (inlineTag.isPresent()
-            && isSummaryTag(inlineTagNode)
-            && isDefinedFirst(inlineTagNode)) {
-            validateSummaryTag(inlineTagNode);
-        }
-        else if (inlineTag.isPresent() && isInlineReturnTag(inlineTagNode)) {
-            validateInlineReturnTag(inlineTagNode);
+        if (inlineTag.isPresent()) {
+            final DetailNode inlineTagNode = inlineTag.get();
```

Comment:
from modernizer: `SummaryJavadocCheck.java:215: Prefer java.util.Optional.orElseThrow`

I mam not sure why we should do it here as we did `inlineTag.isPresent()` looks like false positive in modernizer.

Reply:
suppressed by validator - https://github.com/gaul/modernizer-maven-plugin/blob/a553adf84d6b16b9e18f3d1365afc68cea950fb5/modernizer-maven-plugin/src/main/resources/modernizer.xml#L1293C13-L1293C56